i18n/troubleshooting_es.md

```diff
@@ -0,0 +1,28 @@
+---
+# Ups, algo no está funcionando.
+# ¡No desesperes!  
+---
+
+## Comprueba tu solución:
+
+`Solución
+===================`
+
+%solution%
+
+`Tu intento
+===================`
+
+%attempt%
+
+`Diferencia
+===================`
+
+%diff%
+
+## Ayuda adicional:
+ * Has escrito el nombre del archivo correctamente? Puedes comprobarlo ejecutando ls `%filename%`, si tu ves ls: cannot access `%filename%`: No such file or directory entonces deberías crear un nuevo archivo o cambiar de directorio al que contiene tu archivo
```

Comment:
@esalinas00 is gramatically correct. that being said, I'm in favor of following a convention: every question with both question marks or every question with only one at the end.